src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ShowColumns.txt

```diff
@@ -0,0 +1,31 @@
+﻿#SETUP: SupportColumnNamesAsIdentifiers
+
+>> ShowColumns(Table({a:1,b:2},{a:3,b:4}), b)
+Table({b:2},{b:4})
+
+>> ShowColumns(Table({a:1,b:2},{a:3,b:4}), b, a)
+Table({a:1,b:2},{a:3,b:4})
+
+>> ShowColumns(ShowColumns(Table({a:1,b:2},{a:3,b:4}), a), a)
+Table({a:1},{a:3})
+
+>> ShowColumns(AddColumns(Table({a:1,b:2},{a:3,b:4}), c, a * b), c)
+Table({c:2},{c:12})
+
+>> AddColumns(ShowColumns(Table({a:1,b:2},{a:3,b:4}), a), c, a * a * a)
+Table({a:1,c:1},{a:3,c:27})
+
+>> ShowColumns(Table({a:1,b:2},{a:3,b:4}), a, c)
+Errors: Error 43-44: The specified column 'c' does not exist. The column with the most similar name is 'a'.|Error 0-45: The function 'ShowColumns' has some invalid arguments.
+
+>> Last(ShowColumns(Table({a:1,b:2},{a:3,b:4}), b)).b
```

Comment:
>Last(ShowColumns(Table({a:1,b:2},{a:3,b:4}), b)).b [](http://example.com/codeflow?start=3&length=50)

It would be good to have a negative test if we try to access `.a` instead of `.b`